src/mpi/coll/ialltoallv/Makefile.mk

```diff
@@ -18,4 +18,6 @@ mpi_sources +=                                    \
 mpi_core_sources +=                               \
     src/mpi/coll/ialltoallv/ialltoallv_intra_inplace.c  \
     src/mpi/coll/ialltoallv/ialltoallv_intra_blocked.c  \
-    src/mpi/coll/ialltoallv/ialltoallv_inter_pairwise_exchange.c
+    src/mpi/coll/ialltoallv/ialltoallv_inter_pairwise_exchange.c  \
+    src/mpi/coll/ialltoallv/ialltoallv_gentran_algos.c  \
+    src/mpi/coll/ialltoallv/ialltoallv_intra_gentran_scattered.c
```

Comment:
We tried to put them as original MPICH algorithms first and then new ones at the bottom, but this can be changed if you insist. 
Like you said if we change it here it will need to be changed for all algorithms. Can we make a separate commit for it once all the algorithms are merged or maybe do them as they are merged. It doesn't make sense to change this for all algorithms here since we have outstanding PRs where the same feedback would apply.

Reply:
Agreed (we can leave for a separate PR at the end).

Reply:
I don't think consistency between files is all that important in this case. If we want to keep things ordered in a particular Makefile (and whenever we add new ones), that makes sense.

Basically, I don't think it's necessary to change all other Makefiles as part of this PR.